### ISSUE TITLE
About cfg_files and pretrained resnet50 model.

### ISSUE THREAD

**Issue by YuanYeshang** [minhnhatvt/label-distribution-learning-fer-tf#5]
Hello, I want to know that if I want to train the model on affectnet with resnet50, what cfg_file should I use? As I see, there is no cfg_file for affnet on resnet50, just on resnet18. So could you please share the cfg_file for affectnet on resnet50 and the pretrained resnet50 with input size 224*224? I am doing my graduation design based on your work. I will appreciate if I can get your help. Thank you.

**Comment by YuanYeshang** [comment]
In addition, what cfg_file should I use if I want to train the model on SFEW?

**Comment by YuanYeshang** [comment]
Besides, how do you test the model on sfew dataset as the test set don't have labels. Did you test it on the val set?
